libcxx/docs/Status/Cxx23Papers.csv

```diff
@@ -104,9 +104,9 @@
 "`P2708R1 <https://wg21.link/P2708R1>`__","LWG", "No Further Fundamentals TSes", "November 2022","|Nothing to do|","",""
 "","","","","","",""
 "`P0290R4 <https://wg21.link/P0290R4>`__","LWG", "``apply()`` for ``synchronized_value<T>``","February 2023","","","|concurrency TS|"
-"`P2770R0 <https://wg21.link/P2770R0>`__","LWG", "Stashing stashing ``iterators`` for proper flattening","February 2023","","","|ranges|"
+"`P2770R0 <https://wg21.link/P2770R0>`__","LWG", "Stashing stashing ``iterators`` for proper flattening","February 2023","|Partial| [#note-P2770R0]_","","|ranges|"
```

Comment:
I was looking into implementing `join_with` but since you arelady did such a good job, are you planning also to finish the `join_view` part as that view is marked experimental as it hasn't complete these changes?

Reply:
Yes, I'm working on it too.

Reply:
@JMazurkiewicz Great! You should probably update the status page that you are working on these items:
https://github.com/llvm/llvm-project/blob/main/libcxx/docs/Status/RangesViews.csv
https://libcxx.llvm.org/Status/Ranges.html

Although for some reason it doesn't get rendered right now. I guess it is a CI issue. BTW. You can use a VSCode extension to check if rst files are rendered correctly.